Network/HTTP/Req.hs

```diff
@@ -861,6 +871,19 @@ instance HttpMethod method => RequestComponent (Womb "method" method) where
 data Url (scheme :: Scheme) = Url Scheme (NonEmpty Text)
   -- NOTE The second value is the path segments in reversed order.
   deriving (Eq, Ord, Show, Data, Typeable, Generic)
+type role Url nominal
+
+-- With template-haskell >=2.15 and text >=1.2.4 Lift can be derived, however
+-- the derived lift forgets the type of scheme.
+instance Typeable scheme => TH.Lift (Url scheme) where
+  lift url =
+    TH.dataToExpQ (fmap liftText . cast) url `TH.sigE` case url of
+      Url Http _  -> [t|Url 'Http|]
+      Url Https _ -> [t|Url 'Https|]
+    where liftText t = TH.AppE (TH.VarE 'T.pack) <$> TH.lift (T.unpack t)
+#if MIN_VERSION_template_haskell(2,16,0)
```

Comment:
`ormolu` doesn't like `CPP`. Not implementing `liftTyped` wouldn't break our code.

Reply:
Use latest version of Ormolu.